packages/create-atomic/src/plopfile.ts

```diff
@@ -77,21 +77,21 @@ export default function (plop: NodePlopAPI) {
         ],
       },
       function installPackagesPrompt() {
-        return 'Installing packages 🚀';
+        return '🚀  Installing packages...';
       },
       function installPackages(answers) {
         return new Promise((resolve, reject) => {
           const {project} = answers as PromptsAnswers;
           const process = spawn(getPackageManager(), ['install'], {
-            stdio: 'inherit',
+            stdio: 'ignore',
```

Comment:
I still see a loading indicator, but no more bash vomit

Reply:
Ok cool! If there's a loading indicator, we're good :)